roles/cobbler/templates/triggers/install/post/cephlab_ansible.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+## {{ ansible_managed }}
+set -ex
+name=$2
+export USER=root
+export HOME=/root
+ANSIBLE_CM_PATH=/root/ceph-cm-ansible
+
+# Bail if the ssh port isn't open.
+nc -vz $name 22
+
+mkdir -p /var/log/ansible
+
+pushd $ANSIBLE_CM_PATH
+ANSIBLE_SSH_PIPELINING=1 ansible-playbook testnodes.yml -vv --limit $name* --tags user,pubkeys 2>&1 | tee /var/log/ansible/$name.log
```

Comment:
Once we merge https://github.com/ceph/ceph-cm-ansible/pull/1 I think the users.yml playbook would be perfect to use here.  It essentially does what --tags user,pubkeys is doing but will work all for all roles.